### ISSUE TITLE
Sulfur-containing abscesss

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9947]
**Preferred term label:**
Sulfur-containing abscesss

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Actinomycosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000458
- label: Sulfur-containing abscess
- parents: [Abscess (HP:0025615)](https://hpo.jax.org/app/browse/term/HP:0025615)
- pmid: PMID:29489261
- def: An abscess containing tiny yellow clumps (sulfur granules), formed by infection with Actinomyces and generally associated with a granulomatous and suppurative infection.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9947
- synonyms:
